.ci_scripts/run_build.sh

```diff
@@ -11,8 +11,17 @@ case "$BUILD_TARGET" in
 	cd build && make && make test && make install
 	echo "Creating disk image"
 	hdiutil create -volname Julius -srcfolder julius.app -ov -format UDZO julius.dmg
+	cd ..
```

Comment:
Why do we want to cd back up here?

Reply:
Otherwise `package_appimage.sh` would start in another directory than this script, which would be confusing.
See https://github.com/bvschaik/julius/blob/6f4c95ed9bcaf419788ac5d675f9f1fb1a8eaa43/.travis.yml#L54-L58

Reply:
No. The directory is changed for the .sh file only. After that's done, you end up in the same directory as where you started, regardless of what the .sh script has done to the working directory.

Reply:
I stand corrected then, it can be removed.